ZeroWasteFriendly/Scenes/ActivityInfo/ActivityInfoPresenter.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ActivityInfoPresenter.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 08/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
try using Generic presenter just like in other project so u don’t implement spinner and alert stuff everywhere